### DISCUSSION TITLE
Where to start

### DISCUSSION THREAD

**Discussion by sayashraaj** [webpack/webpack#12273] [Q&A] [answered]
Hi, I want to contribute to webpack, but I don't know where to start. The codebase is huge and looks intimidating. Could you please guide me?

**Comment by alexander-akait** [comment, marked as answer]
You can chose any https://github.com/webpack/webpack/issues and send a PR, what exactly do you mean by start?

**Reply by sayashraaj** [reply]
"Start" means start understanding the codebase. I am finding it hard to understand the codebase, which is why I am asking for help

**Reply by alexander-akait** [reply]
This is a difficult answer, because there is a lot of different code here, which specifically interests you, unfortunately we cannot take and tell everything about the code
